data-model/fdo-types/0.3.0/digital-specimens/schema/digital-specimen.json

```diff
@@ -0,0 +1,372 @@
+{
+  "$id": "https://schemas.dissco.tech/schemas/digitalobjects/0.3.0/digital-specimens/digital-specimen.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$comment": "DigitalObject Version 0.3.0",
+  "type": "object",
+  "properties": {
+    "ods:id": {
+      "type": "string",
+      "description": "The unique digital identifier of the object",
+      "pattern": "https:\/\/doi.org\/10.22\/(.){3}-(.){3}-(.){3}",
```

Comment:
missing escapes on the periods, should be:

`https:\/\/doi\.org\/10\.22\/(.){3}-(.){3}-(.){3}`